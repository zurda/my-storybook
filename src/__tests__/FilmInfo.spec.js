import React from 'react'
import { create } from 'react-test-renderer'
import FilmInfo from '../components/FilmInfo'

const filmDetails = {
  overview: 'some overview',
  poster_path: '/eufF3xEif5EZ1gTjdubw3KWh9xU.jpg',
  tagline: 'same as overview'
}

describe('Film Info component', () => {
  test('Matches the snapshot', () => {
    const filmInfo = create(<FilmInfo film={filmDetails} />)
    expect(filmInfo.toJSON()).toMatchSnapshot()
  })
})
