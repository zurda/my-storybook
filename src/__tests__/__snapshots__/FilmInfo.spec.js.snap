// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film Info component Matches the snapshot 1`] = `
<div
  className="css-9s8wn8"
>
  <img
    alt="Film poster"
    className="css-rh3cin"
    src="http://image.tmdb.org/t/p/w154/eufF3xEif5EZ1gTjdubw3KWh9xU.jpg"
  />
  <p
    className="css-bni6fn"
  >
    some overview
  </p>
</div>
`;
